core/chains/evm/config/chain_specific_config.go

```diff
@@ -267,6 +267,15 @@ func setChainSpecificConfigDefaultSets() {
 	avalancheFuji := avalancheMainnet
 	avalancheFuji.linkContractAddress = "0x0b9d5D9136855f6FEc3c0993feE6E9CE8a297846"
 
+	// Harmony
+	harmonyMainnet := fallbackDefaultSet
+	harmonyMainnet.linkContractAddress = "0x218532a12a389a4a92fC0C5Fb22901D1c19198aA"
+	harmonyMainnet.gasPriceDefault = *big.NewInt(5000000000)
```

Comment:
Is this Gwei? You can use `assets.GWei(5)` to make it clearer.

Reply:
good call, thank you!